stages/org.osbuild.authselect

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+"""
+Select system identity and authentication sources with authselect.
+
+Sets system identity and authentication sources.
+
+The stage calls `authselect select` to set authselect profile to 'profile_id'.
+Optionally a list of profile features to enable may be provided using 'features'
+option. The list of available profile features can be obtained by running
+`authselect list-features <profile_id>`.
+
+Notes:
+  - Requires 'chroot' in the buildroot.
+  - Runs the 'authselect' binary from the image in the chroot.
+"""
+
+
+import subprocess
+import sys
+
+import osbuild.api
+
+
+SCHEMA = """
+"additionalProperties": false,
+"required": ["profile_id"],
+"description": "Select system identity and authentication sources.",
+"properties": {
+  "profile_id": {
```

Comment:
I think we could just use `profile` here. 

(Sorry for commenting on a merged PR, had no chance to look at this yesterday)

Reply:
I don't mind changing it in a new PR.

Reply:
I'll look at this after my vacation...

Reply:
submitted https://github.com/osbuild/osbuild/pull/729